taxi/urls.py

```diff
@@ -16,8 +22,26 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>/update",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturers/<int:pk>/delete",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete",
+    ),
```

Comment:
Don't forget to add `/` at the end of endpoints.
```suggestion
    path(
        "manufacturers/create/",
        ManufacturerCreateView.as_view(),
        name="manufacturer-create",
    ),
    path(
        "manufacturers/<int:pk>/update/",
        ManufacturerUpdateView.as_view(),
        name="manufacturer-update",
    ),
    path(
        "manufacturers/<int:pk>/delete/",
        ManufacturerDeleteView.as_view(),
        name="manufacturer-delete",
    ),
```